app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,48 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+  padding-bottom: govuk-spacing(8);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: "desktop") {
+    padding-bottom: govuk-spacing(8);
+  }
+}
+
+.homepage-search {
+  @include govuk-media-query($until: "tablet") {
+    max-width: 420px;
+  }
+}
+
+.homepage-most-viewed-list {
+  list-style: none;
+  margin: 0;
+  padding: 0 0 govuk-spacing(7) 0;
+
+  @include govuk-media-query($from: "tablet") {
+    padding-bottom: 0;
+  }
+
+  & > li {
+    padding-bottom: govuk-spacing(4);
+
+    @include govuk-media-query($from: "tablet") {
+      padding-bottom: govuk-spacing(2);
+    }
+  }
+}
+
+.homepage-most-viewed-list__item {
+  font-size: 16px;
+  font-size: govuk-px-to-em(16);
+  font-weight: bold;
+
+  @include govuk-media-query($from: desktop) {
+    font-size: 19px;
```

Comment:
We've been avoiding using `govuk-font` in a few places recently, but this feels like an appropriate use case for `govuk-font`.

Reply:
Good spot - it's a nice mixin that fits this use perfectly. Updated in https://github.com/alphagov/frontend/pull/3028/commits/b4b96872f55821b03d9cd910b429c43fda4fa11a.